recipes/fuse/fuse_2.9.4.oe

```diff
@@ -1,6 +1,7 @@
 require fuse.inc
 
-SRC_URI += "file://dont-run-updaterc.d-on-host.patch"
+SRC_URI += "file://dont-run-updaterc.d-on-host.patch \
+            file://fuse_kernel.h_clean_includes.patch"
```

Comment:
Maybe. It's seems the upstream is in a messy state...
https://github.com/libfuse/libfuse the patch is not applied
http://git.baserock.org/cgit/delta/fuse.git/ the patch is applied

I do not have time to upgrade fuse.